themes/hugo_theme_robust/static/css/styles.css

```diff
@@ -40,17 +40,24 @@ caption {
   line-height:1.5em;
 }
 
-.hljs-string{
-  color:#183691;
+.hljs-number{
+  color:#aa5146  !important;
 }
 
-.hljs-variable{
-  color:#a71d61;
+.hljs-value,
+.hljs-string{
+  color:#183691  !important;
 }
 
+.hljs-title,
+.hljs-tag,
+.hljs-keyword,
+.hljs-variable,
+.hljs-attribute,
+.hljs-attr_selector,
 .hljs-built_in,
 .hljs-preprocessor{
-  color:#000;
+  color:#000  !important;;
```

Comment:
double semicolon probably not what you want here.

Reply:
Thanks for catching that! Unfortunately made no difference to the title vs comment issue.